pyspecProcScripts/integral_w_error.py

```diff
@@ -30,7 +30,11 @@ def integral_w_errors(s,sig_path,error_path, indirect='vd', direct='t2',fl=None,
              not included in the signal pathway.
     """
     assert s.get_ft_prop(direct), "need to be in frequency domain!"
-    frq_slice = integrate_limits(select_pathway(s,sig_path),fl=fl)
+    s.ift('t2')
+    frq_slice = integrate_limits(select_pathway(s,sig_path),
+            convolve_method='Gaussian',
+            fl=None)
+    s.ft('t2')
```

Comment:
I'm confused by this -- as we defined it now, isn't the signal supposed to be in teh frequency domain when you call "integrate_limits"?

Reply:
actually this breaks the check_integration_error example, so I'm doing to roll it back.

Reply:
rather, I leave the central function call alone, but remove the bracketing expressions that put it in the time domain.

Reply:
yes it should be in the frequency domain when given to integrate_limits - I had not touched integral_w_error for some time and was mostly focused on integrate_limits, so I think that just got missed (I hadn't been running check_integration_error as one of the example scripts)